cpp/cmake/Dependencies.cmake

```diff
@@ -16,6 +16,36 @@
 
 include(ExternalProject)
 
+##############################################################################
+# - libcudacxx
+
+set(CUDA_CXX_DIR ${CMAKE_CURRENT_BINARY_DIR}/libcudacxx CACHE STRING "Path to libcudacxx repo")
```

Comment:
This is no longer the file where we get dependencies from. RAFT already has a get_cuco.cmake file for u collections, it would be very easy to do a similar one for libcudacxx and add both for cuML (can be done in this PR, or I can help with another quick PR as well)